server/tests/controllerTests/user.test.js

```diff
@@ -0,0 +1,36 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import app from '../../../app';
+import data from '../utilities/mockData';
```

Comment:
We're not using a test db anymore?

Reply:
@seyi-adeleke we are. The mockData is just test data I'm sending to the db;